.github/workflows/tests.yml

```diff
@@ -41,11 +41,7 @@ jobs:
       with:
```

Comment:
REVIEWERS: This PR also removes the algol60 dependency in the workflows since this package is supposed to be independent of HPE resources.